Client/src/js/views/SchoolBuses.jsx

```diff
@@ -273,14 +273,14 @@ var SchoolBuses = React.createClass({
   },
 
   sendEmail(email){
+    this.closeEmailDialog();
```

Comment:
This will introduce an issue if the sendEmail function doesn't return a response (i.e. timeout), or if it returns a non-200 response code.

Expected result will be the Email Dialog just closing.  This should be resolved before going to PROD.